common/src/main/resources/data/farm_and_charm/crafting_bowl/corn_flour.json

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "farm_and_charm:crafting_bowl",
+  "ingredients": [
+    {
+      "item": "cornexpansion:cornmeal"
+    },
+    {
+      "item": "farm_and_charm:flour"
```

Comment:
Unsure about the usage of regular flour to make corn flour
According to the internet, corn flour is made by "grinding dried corn kernels", while cornmeal is made by "grinding regular corn kernels'. Should we reflect that in the mod (ideally, we would add enough use cases for both the flour and the cornmeal for them to not be a crafting step but a meaningful recipe), or keep things simple and use another method ? (not sure how we would differentiate them without dried kernels tho)

Reply:
Kernels as seeds I kinda view as dried to begin with. I think it makes sense to change this to reflect reality. And yeah, would love to bring in more ideas to use corn dough (I'm thinking pancakes at a minimum, maybe corn dogs as another thought) but for cornmeal that's forward thinking for pizza lol.